### ISSUE TITLE
VS_DIST_TO_EMPTY and VS_VEHICLE_STATE are not always available in the response

### ISSUE THREAD

**Issue by ms1995** [G-Two/subarulink#58]
This happened in my case. Here's the raw response I got:

```
{'data': {'avgFuelConsumptionLitersPer100Kilometers': None,
          'avgFuelConsumptionMpg': None,
          'distanceToEmptyFuelKilometers': None,
          'distanceToEmptyFuelKilometers10s': None,
          'distanceToEmptyFuelMiles': None,
          'distanceToEmptyFuelMiles10s': None,
          'evDistanceToEmptyByStateKilometers': None,
          'evDistanceToEmptyByStateMiles': None,
          'evDistanceToEmptyKilometers': None,
          'evDistanceToEmptyMiles': None,
          'evStateOfChargePercent': None,
          'eventDate': 1664270191000,
          'eventDateStr': '2022-09-27T09:16+0000',
          'latitude': None,
          'longitude': None,
          'odometerValue': REDACTED,
          'odometerValueKilometers': REDACTED,
          'positionHeadingDegree': None,
          'tirePressureFrontLeft': None,
          'tirePressureFrontLeftPsi': None,
          'tirePressureFrontRight': None,
          'tirePressureFrontRightPsi': None,
          'tirePressureRearLeft': None,
          'tirePressureRearLeftPsi': None,
          'tirePressureRearRight': None,
          'tirePressureRearRightPsi': None,
          'tyreStatusFrontLeft': None,
          'tyreStatusFrontRight': None,
          'tyreStatusRearLeft': None,
          'tyreStatusRearRight': None,
          'vehicleStateType': None,
          'vhsId': REDACTED,
          'windowFrontLeftStatus': None,
          'windowFrontRightStatus': None,
          'windowRearLeftStatus': None,
          'windowRearRightStatus': None,
          'windowSunroofStatus': None},
 'dataName': None,
 'errorCode': None,
 'success': True}
```

Such response will cause `subarulink/controller.py:944` to throw an exception.

**Comment by G-Two** [comment]
Thanks for reporting. The situation is now handled in v0.6.1

**Comment by G-Two** [comment]
Fixed in #61
